src/main/java/de/adesso/kicker/login/LoginController.java

```diff
@@ -1,7 +1,9 @@
-package de.adesso.kicker.user;
+package de.adesso.kicker.login;
```

Comment:
The login parts of the code are going to be replaced in the future due to keycloak. This is just going to cause unnecessary conflicts. Please don't do things that aren't of concern to you/the branch.